circle.yml

```diff
@@ -34,7 +34,7 @@ checkout:
 
 test:
   override:
-    - case $CIRCLE_NODE_INDEX in 0) (cd app; meteor npm run test) ;; 1) (cd app; meteor npm run test-app) ;; 2) (cd app; ./.testing/install_chimp.sh; ./tests/acceptance_run) ;; esac:
+    - case $CIRCLE_NODE_INDEX in 0) (cd app; meteor npm run test) ;; 1) (cd app; meteor npm run coverage) ;; 2) (cd app; ./.testing/install_chimp.sh; ./tests/acceptance_run) ;; esac:
```

Comment:
Since `run coverage` includes both unit and app tests, we should have `run coverage` replace `run test` and `run test-app`. It's kind of a bummer because then I can't parallelize them like I have now. 

I wonder if there's a tool that can merge the two lcov reports after the fact? I could have that run as some sort of `post-test` task in CI maybe.